docs/resources/project.md

```diff
@@ -71,65 +74,70 @@ resource "project" "myproject" {
 
 ### Required
 
-- **admin_privileges** (Block Set, Min: 1) (see [below for nested schema](#nestedblock--admin_privileges))
-- **display_name** (String) Also known as project name on the UI
-- **key** (String, ForceNew) The Project Key is added as a prefix to resources created within a Project. This field is mandatory and supports only 3 - 6 lowercase alphanumeric characters. Must begin with a letter. For example: us1a.
+- `admin_privileges` (Block Set, Min: 1) (see [below for nested schema](#nestedblock--admin_privileges))
+- `display_name` (String) Also known as project name on the UI
+- `key` (String) The Project Key is added as a prefix to resources created within a Project. This field is mandatory and supports only 3 - 10 lowercase alphanumeric and hyphen characters. Must begin with a letter. For example: us1a.
```

Comment:
Consider putting `us1a` in a code block to avoid confusion. For example, is the `.` part of the example project key? 

Consider adding a hyphen to the example for a more robust example

Reply:
The original text was copied from Artifactory's tooltip: 
![Screenshot 2022-11-17 at 2 42 30 PM](https://user-images.githubusercontent.com/27263/202575814-b63ee7dd-7fcc-4fa6-bb08-7874bc815159.png)